### ISSUE TITLE
Debug level of log4j is enabled by default

### ISSUE THREAD

**Issue by MikeShysh** [Visual-Regression-Tracker/sdk-java#34]
Hi,

Observed on 4.3.1

Preconditions:
```
    implementation group: 'com.github.visual-regression-tracker', name: 'sdk-java', version: '4.3.1'
    implementation 'net.lightbody.bmp:browsermob-core:2.1.5'
```
com.github.visual-regression-tracker
STR:
1) run test with enabled Proxy

Actual result: lots of Proxy debug messages in the console
![image](https://user-images.githubusercontent.com/13480390/96750859-25519f00-13d5-11eb-8f40-96731f08ed1f.png)
Expected result: debug for proxy is turned off

Additional info: the issue is not observed on 4.3.0 version

**Comment by pashidlos** [comment]
issue was introduced here https://github.com/Visual-Regression-Tracker/sdk-java/pull/32

@alexromanov would you like to take it?

**Comment by alexromanov** [comment]
hi. yes. I'll investigate it and fix.

**Comment by pashidlos** [comment]
fix released in `4.3.2`
